### PR TITLE
Capture Skia trace events

### PR DESCRIPTION
**Pull request by jason-simmons**

### CODE REVIEW

File: sky/shell/shell.cc [hunk 1]
```diff
@@ -149,6 +150,9 @@ void Shell::InitStandalone(std::string icu_data_path) {
 
 void Shell::Init() {
   base::DiscardableMemoryAllocator::SetInstance(&g_discardable.Get());
+  #ifndef FLUTTER_PRODUCT_MODE
+    InitSkiaEventTracer();
```
Comment (abarth) [inline sky/shell/shell.cc]: bad indent

**Comment by abarth** [comment]
LGTM

File: sky/shell/shell.cc (hunk 1 above)
Comment (jason-simmons) [inline sky/shell/shell.cc]: done

**Comment by Hixie** [comment]
Looks like this caused a bot to fail so I'm reverting it.

**Comment by jason-simmons** [comment]
The build was broken due to an unrelated issue.

Will land a fix (https://github.com/flutter/engine/pull/2833) and then reinstate this change

**Pull request merged**